src/index.html

```diff
@@ -11,7 +11,173 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <header>
+      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+
+        <section class="group">
```

Comment:
Don't add empty lines between parent and child elements.
```suggestion
      <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
        <section class="group">
```